### PR TITLE
Polished stop container button and added toggling start/stop button

### PR DESCRIPTION
**Pull request by Elesant**
screen shot 2015-04-27 at 9 31 21 am
screen shot 2015-04-27 at 9 31 48 am

### CODE REVIEW

File: src/components/ContainerDetailsSubheader.react.js [hunk 1]
```diff
@@ -54,12 +54,18 @@ var ContainerDetailsSubheader = React.createClass({
     }
     return (this.props.container.State.Downloading || this.props.container.State.Restarting);
   },
-  stopDisabled: function () {
+  disableStop: function () {
```
Comment (jeffdm) [inline src/components/ContainerDetailsSubheader.react.js]: Rename to stopDisabled?

File: src/components/ContainerDetailsSubheader.react.js [hunk 2]
```diff
@@ -54,12 +54,18 @@ var ContainerDetailsSubheader = React.createClass({
...
     if (!this.props.container) {
       return false;
     }
     return (this.props.container.State.Downloading || this.props.container.State.ExitCode);
   },
+  disableStart: function () {
```
Comment (jeffdm) [inline src/components/ContainerDetailsSubheader.react.js]: Rename to startDisabled?

File: src/stores/ContainerStore.js [hunk 3]
```diff
@@ -411,6 +411,19 @@ var ContainerStore = assign(Object.create(EventEmitter.prototype), {
       }
     });
   },
+  start: function (name, callback) {
+    var container = docker.client().getContainer(name);
+    _muted[name] = true;
```
Comment (jeffdm) [inline src/stores/ContainerStore.js]: No need to mute here, or in Stop. I didn't catch this in the last one

**Comment by jeffdm** [comment]
Nice! Added some comments

File: src/components/ContainerDetailsSubheader.react.js (hunk 2 above)
Comment (Elesant) [inline src/components/ContainerDetailsSubheader.react.js]: @jeffdm tried to keep the name consistent with the other disable methods, should I change the name of everything else?

**Comment by jeffdm** [comment]
LGTM

**Pull request merged**